src/network/networkprotocol.h

```diff
@@ -178,6 +178,7 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 			Breaks compatibility with older clients.
 	PROTOCOL VERSION 36:
 		Backwards compatibility drop
+		Add fullbright to object properties
```

Comment:
Not sure this needs adding here.

Reply:
it should as it's transported on network

Reply:
Ok will add my 'can zoom' then.